### ISSUE TITLE
getting 404 on requesting Hyperlegible fonts

### ISSUE THREAD

**Issue by martenson** [galaxyproject/galaxy#15242]
I am getting two browser 404s on galaxy load using latest `dev`

`http://127.0.0.1:8080/static/style/5dc77fa7c8c5443ffdad.woff`
`http://127.0.0.1:8080/static/style/7c05f33b4fe5f8132439.woff2`

links are coming from this `base.css`:

```css
@font-face{font-display:swap;font-family:Atkinson Hyperlegible;font-style:normal;font-weight:400;src:url(a278964027d51119ea29.woff2) format("woff2"),url(5dc77fa7c8c5443ffdad.woff) format("woff");unicode-range:u+0100-024f,u+0259,u+1e??,u+2020,u+20a0-20ab,u+20ad-20cf,u+2113,u+2c60-2c7f,u+a720-a7ff}@font-face{font-display:swap;font-family:Atkinson Hyperlegible;font-style:normal;font-weight:400;src:url(7c05f33b4fe5f8132439.woff2) format("woff2"),url(5dc77fa7c8c5443ffdad.woff) format("woff");unicode-range:u+00??,u+0131,u+0152-0153,u+02bb-02bc,u+02c6,u+02da,u+02dc,u+2000-206f,u+2074,u+20ac,u+2122,u+2191,u+2193,u+2212,u+2215,u+feff,u+fffd}
```

**Comment by ElectronicBlueberry** [comment]
I've looked a bit into this, but could not find a solution. What I did notice:
* The font works fine. Is it maybe being imported twice?
* I've tried:
   - Importing via scss (same behavior)
   - Importing via scss mixins (same behavior)
   - Modifying import order (same behavior)
   - Using webpack file-loader (broke fontawesome + same behavior)
   - Removing the "@font-source" package, and self hosting the files in "assets/fonts" (font no longer works + same error message)
